src/SonarQube.Client/Models/Versions.cs

```diff
@@ -18,13 +18,10 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using Newtonsoft.Json;
-
-namespace SonarLint.VisualStudio.Integration.Service.DataModel
+namespace SonarQube.Client.Models
 {
-    internal class QualityProfiles
+    public class VersionDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'VersionDTO' to match camel case naming rules, consider using 'VersionDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)